glumpy/library/colormaps/user.glsl

```diff
@@ -8,7 +8,7 @@ uniform sampler1D colormap;
 
 vec3 colormap_user(float t)
 {
-    return texture1D(colormap, t).rgb;
+    return texture(colormap, t).rgb;
```

Comment:
Why this change?

Reply:
Was not supposed to be here. I thought it would remain in my fork.

Reply:
Still learning git, sorry.

Reply:
It is there to correct an error (not a warning) for being deprecated since glsl 130.

Reply:
So, sorry for the confusion. I think I moved it back to what it used to be.